### DISCUSSION TITLE
New Section in website😎

### DISCUSSION THREAD

**Discussion by sudhanshutech** [GDSCITM/GDSCITM-official-website#117] [Announcements]
We are coming with a new section design(UI) in this week especially for contributors🎉. So get ready to build the section😁

**Comment by Yashsharma1911** [comment]
Excited to see that section 😎

**Comment by anshitmishra** [comment]
i have added a new gallery section check if any changes and let me know

**Reply by Yashsharma1911** [reply]
@anshitmishra do you know why the gallery section is not showing images? in production

![Screenshot 2023-04-12 at 11 42 52 AM](https://user-images.githubusercontent.com/71271069/231366332-ad849c1f-c2b2-41e6-a3ac-d93ae5605249.png)

**Reply by anshitmishra** [reply]
![Screenshot (49)](https://user-images.githubusercontent.com/91340109/231393329-1758dfb9-b0c1-44f5-88c8-543b15893058.png)


@Yashsharma1911 it showing to me. can you explain the issue, in brief, I didn't understand the issue
